src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/GrunnlagsdataRegisterService.kt

```diff
@@ -24,26 +24,49 @@ class GrunnlagsdataRegisterService(
         personIdent: String,
         barneforeldreFraSøknad: List<String>,
     ): GrunnlagsdataDomene {
-        val pdlSøker = personService.hentSøker(personIdent)
-        val pdlBarn = hentPdlBarn(pdlSøker)
-        val barneForeldre = hentPdlBarneForeldre(pdlBarn, personIdent, barneforeldreFraSøknad)
-        val tidligereVedtasksperioderAnnenForelder = hentTidligereVedtaksperioderAnnenForelder(barneForeldre)
-        val dataTilAndreIdenter = hentDataTilAndreIdenter(pdlSøker)
-
-        val medlUnntak = personopplysningerIntegrasjonerClient.hentMedlemskapsinfo(ident = personIdent)
-
+        val grunnlagsdataFraPdl = hentGrunnlagsdataFraPdl(personIdent, emptyList())
+        val medlUnntak = personopplysningerIntegrasjonerClient.hentMedlemskapsinfo(personIdent)
         val tidligereVedtaksperioder =
-            tidligereVedaksperioderService.hentTidligereVedtaksperioder(pdlSøker.folkeregisteridentifikator)
+            tidligereVedaksperioderService.hentTidligereVedtaksperioder(grunnlagsdataFraPdl.søker.folkeregisteridentifikator)
+        val tidligereVedtasksperioderAnnenForelder = hentTidligereVedtaksperioderAnnenForelder(grunnlagsdataFraPdl.barneForeldre)
 
         return GrunnlagsdataDomene(
-            søker = mapSøker(pdlSøker, dataTilAndreIdenter),
-            annenForelder = mapAnnenForelder(barneForeldre, tidligereVedtasksperioderAnnenForelder),
+            søker = mapSøker(grunnlagsdataFraPdl.søker, grunnlagsdataFraPdl.andrePersoner),
+            annenForelder = mapAnnenForelder(grunnlagsdataFraPdl.barneForeldre, tidligereVedtasksperioderAnnenForelder),
             medlUnntak = medlUnntak,
-            barn = mapBarn(pdlBarn),
+            barn = mapBarn(grunnlagsdataFraPdl.barn),
             tidligereVedtaksperioder = tidligereVedtaksperioder,
         )
     }
 
+    fun hentGrunnlagsdataUtenVedtakshitorikkFraRegister(
+        personIdent: String,
+    ): GrunnlagsdataDomene {
+        val grunnlagsdataFraPdl = hentGrunnlagsdataFraPdl(personIdent, emptyList())
+        val medlUnntak = personopplysningerIntegrasjonerClient.hentMedlemskapsinfo(personIdent)
+        return GrunnlagsdataDomene(
+            søker = mapSøker(grunnlagsdataFraPdl.søker, grunnlagsdataFraPdl.andrePersoner),
+            annenForelder = mapAnnenForelder(grunnlagsdataFraPdl.barneForeldre, emptyMap()),
+            medlUnntak = medlUnntak,
+            barn = mapBarn(grunnlagsdataFraPdl.barn),
+            tidligereVedtaksperioder = null,
+        )
+    }
```

Comment:
Gir det mest mening å spesifisere når noe er _uten_ noe enn _med_ noe?

Reply:
Godt poeng - frem til nå har metoden for å hente grunnlagsdata inkludert vedtakshistorikk. Det er jo av interesse når vi skal lagre grunnlagsdata for en behandling, men er ikke interessant for å vise personopplysninger... 

Det som gjør mest vondt er at begge returnerer samme objekt - som kan inneholde vedtakshistorikk... :/